### PR TITLE
update leverages. remove one

### PR DESCRIPTION
**Pull request by ermalkaleci**

### CODE REVIEW

File: modules/primitives/src/lib.rs [hunk 1]
```diff
@@ -61,22 +61,20 @@ impl Convert<Price, Balance> for BalancePriceConverter {
...
-		ShortThirty	= 0b0100000000000000,
-		ShortFifty	= 0b1000000000000000,
+		LongTwo 	= 0b0000000000000001,
+		LongThree 	= 0b0000000000000010,
+		LongFive 	= 0b0000000000000100,
+		LongTen 	= 0b0000000000001000,
+		LongTwenty 	= 0b0000000000010000,
+		LongThirty	= 0b0000000000100000,
+		LongFifty	= 0b0000000001000000,
+		ShortTwo 	= 0b0000000010000000,
```
Comment (xlc) [inline modules/primitives/src/lib.rs]:
```suggestion
		ShortTwo 	= 0b0000000100000000,
```

I want first 8 bits for long and last 8 bits for short so short to long is `>> 8` and long to short is `<< 8`

Comment (ermalkaleci) [reply modules/primitives/src/lib.rs]: @xlc ok

Comment (ermalkaleci) [reply modules/primitives/src/lib.rs]: @xlc updated tests to make sure you never it covers everything

File: modules/synthetic-protocol/src/lib.rs [hunk 2]
```diff
@@ -179,7 +179,7 @@ impl<T: Trait> Module<T> {
 		);
 
 		ensure!(
-			T::LiquidityPools::is_allowed_position(pool_id, currency_id, Leverage::LongOne),
+			T::LiquidityPools::is_allowed_position(pool_id, currency_id, Leverage::LongTwo),
```
Comment (xlc) [inline modules/synthetic-protocol/src/lib.rs]: this isn't right, but you can fix it in next PR by change this to use `can_mint`

Comment (ermalkaleci) [reply modules/synthetic-protocol/src/lib.rs]: yes that's the plan

**Pull request merged**